mycroft/skills/main.py

```diff
@@ -41,6 +42,7 @@
 event_scheduler = None
 skill_manager = None
 
+DEBUG = Configuration.get().get("debug", False)
```

Comment:
Shouldn't we add this to `mycroft.conf`?

Reply:
Ok, mycroft.conf is now updated. You're probably right that it's good to have in the config as a reference.